.stylelintrc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why is this file duplicated from the same file in [the fluid-grunt-lint-all pull](https://github.com/fluid-project/fluid-grunt-lint-all/pull/20/files)?  Is there any way to reused or `extend` the configuration used in that package instead?

Reply:
I think much in the way that the Fluid eslint configuration is installed as a package which is extended within Infusion's `.eslintrc` file, we'd need to create and publish a package (e.g. `stylelint-config-fluid`) which could be extended in `.stylelintrc`.

Reply:
I was thinking we could resolve the path to one stored in fluid-grunt-lint-all, but I think your proposal is better given that we're trending towards improving support for (and presumably relying more on) central configurations like this.  

Can you file a ticket for that work?  I imagine you'd probably be best to write it up slash work on the actual thing.

I was going to say that we could do that later and avoid having three related pulls open at once, but I think any change to a central config is going to necessitate at least a small pull against fluid-grunt-lint-all and infusion.  If it's not too much work, I think it'd be cleaner to have a common config as part of the initial effort to standardise our approach to CSS.

Reply:
@the-t-in-rtf I haven't filed an issue (not sure where I should do that, to be honest) but I started working on this in https://github.com/greatislander/stylelint-config-fluid. I'll try to get it wrapped up this week.